workshop/exercise-4/README.md

```diff
@@ -44,22 +44,22 @@ Because Knative is built on top of Kubernetes, you can use kubectl along with a
 
     Note: To exit the watch, use `ctrl + c`.
 
-4. Let's try out our new application! First, we'll need to know the external IP for our service. Run the following command, and note the value for `EXTERNAL-IP`.
+4. Let's try out our new application! First, let's get the domain name that Knative assigned to the Service we just deployed. Run the following command, and note the value for `domain`. IBM Cloud Kubernetes Service sets the default domain name for Knative to match the domain name of your IBM Cloud Kubernetes Service cluster. It will also set up the Istio ingress to route all incoming requests targeted to that domain to Knative.
 
     ```
-    kubectl get svc istio-ingressgateway --namespace istio-system
+    kubectl get ksvc fib-knative
     ```
 
-5. We will also need to know the domain name that Knative assigned to the Service we just deployed. Run the following command, and note the value for `domain`.
+5. The url should look something like `fib-knative.default.bmv-kubeflow.us-south.containers.appdomain.cloud`. We can set an environment variable so that we can use this throughout the lab:
 
     ```
-    kubectl get ksvc fib-knative
+    export MY_URL=<your_app_url_here>
```

Comment:
I really don't like that Kn uses the term "domain", I would prefer "url" but since they use "domain" in the output and you call it "domain" above, this should probably be "your_app_domain_here".

Reply:
Oh, and MY_URL might need to be MY_DOMAIN, or just MY_APP if we want to avoid it  :-)

Reply:
okay, done